temporalio/client.py

```diff
@@ -1165,29 +1165,68 @@ async def report_cancellation(self, *details: Any) -> None:
         )
 
 
-class WorkflowDescription:
-    """Description for a workflow."""
-
-    def __init__(
-        self,
-        raw_message: temporalio.api.workflowservice.v1.DescribeWorkflowExecutionResponse,
-    ):
-        """Create a workflow description from a describe response."""
-        self._raw_message = raw_message
-        status = raw_message.workflow_execution_info.status
-        self._status = WorkflowExecutionStatus(status) if status else None
+@dataclass
+class WorkflowExecutionDescription:
+    """Description for a workflow execution."""
```

Comment:
Could use a comment on each of the fields here, we should've done the same for TS @lorensr

Reply:
Updated